node-packages/commons/src/tasks.ts

```diff
@@ -907,27 +877,14 @@ export const createPromoteTask = async function(promoteData: any) {
     // type,
   } = promoteData;
 
-  const project = await getActiveSystemForProject(projectName, 'Promote');
+  const project = await getOpenShiftInfoForProject(projectName);
 
-  if (typeof project.activeSystemsPromote === 'undefined') {
-    throw new UnknownActiveSystem(
-      `No active system for tasks 'deploy' in for project ${projectName}`
-    );
-  }
-
-  switch (project.activeSystemsPromote) {
-    case 'lagoon_controllerBuildDeploy':
-        // use deployTargetPromote function to handle
-        let lagoonData = {
-          projectId: project.id,
-          promoteData
-        }
-        return deployTargetPromote(lagoonData)
-    default:
-      throw new UnknownActiveSystem(
-        `Unknown active system '${project.activeSystemsPromote}' for task 'deploy' in for project ${projectName}`
-      );
+  // use deployTargetPromote function to handle
+  let lagoonData = {
+    projectId: project.id,
```

Comment:
This needs to be `projectId: project.project.id` because `getOpenShiftInfoForProject`. See other comment on possible resolution